projects/sd-jwt-dotnet.md

```diff
@@ -0,0 +1,68 @@
+# Project Name
+
+SD-JWT-DotNet
+
+# Preferred Maturity Level
+
+Labs
+
+# Project Description
+
+A Dot Net implementation of the [Selective Disclosure for JWTs](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-04.html) (SD-JWT) specification.
+
+# Alignment with the OpenWallet Foundation Mission
+
+This library implements the Selective Disclosure for JWTs specification, which is an important building block for future digital identity wallets.
+
+# Code of Conduct
+
+None
+
+# TAC Sponsor
+
+None
+
+# Project License
+
+Apache 2.0.
+
+# Source Control
+
+[https://github.com/thomas-tran/sd-jwt-dotnet](https://github.com/thomas-tran/sd-jwt-dotnet)
```

Comment:
I would suggest a bit more detailed readme that at least provides some steps on how to set up the project and use the library (snippets). Similarly to the examples that follows:
* https://github.com/openwallet-foundation-labs/wallet-framework-dotnet#quickstart-guide
* https://github.com/openwallet-foundation-labs/sd-jwt-kotlin#checking-out-the-implementation
* https://github.com/openwallet-foundation-labs/sd-jwt-python#setup

Reply:
I am working on the documentation now. I will update shortly

Reply:
It is done in the develop branch